src/frontend/package.json

```diff
@@ -17,7 +17,7 @@
     "watch-ts": "yarn build-ts --watch",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "build-theme": "cunningham -g scss -o scss"
+    "build-theme": "cunningham -g scss -o scss && cunningham -g css -o scss/css"
```

Comment:
The point here is to have two differents files path for `cunningham-tokens.scss` and `cunningham-tokens.css` in order to import them like: 
```scss
@import 'my-file';
@import 'whatever/my-file';
```
in `main.scss`.

The reason is that import with extention (like `.css`) will load a second file in the browser instead of merging it.
With 
```scss
@import 'my-file.scss';
@import 'my-file.css';
```
The `.css` file is not include in our styles and cunningham tokens aren't override.

Reply:
Yup agreed with this point.
But instead to create a dedicated folder for Cunningham, I think it's better to create a `vendors` folder and put all related package stylesheets in this one.

About the import issue you mention, is it problematic to load two stylesheets ?
If yes, do you try the `@use` sass directive in place of `@import` ?